src/nebulr/nebulr-config/nebulr-config.module.ts

```diff
@@ -85,6 +86,16 @@ export class NebulrConfigModule {
       imports.push(
         GraphQLModule.forRoot({
           debug: true,
+          formatError: (error: GraphQLError) => {
+            const errorObj = JSON.parse(error.message);
+            const graphQLFormattedError = {
+              nblocksError: errorObj.statusCode || error.extensions?.code || error.name,
+              message: errorObj.message || error.extensions?.exception?.response?.message,
+              errorCode: errorObj.error.toUpperCase() || error.extensions?.code || "SERVER_ERROR"
+            };
+            console.log(graphQLFormattedError);
```

Comment:
If this makes the Exception to show in the terminal it's a very good improvement!
Right now the graphql errors are silent and just passes through, even if its a internal validation error due to bad query (not error in account-api). Will those show up here aswell?

Change to `console.error(graphQLFormattedError);` to use the correct console output.

Reply:
renamed